BlazarUI/app/scripts/components/shared/PageHeader.jsx

```diff
@@ -1,10 +1,41 @@
 import React, {Component, PropTypes} from 'react';
 
-class PageHeader extends Component{
+class PageHeader extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      headerClass: '',
+      isSmall: false
```

Comment:
lets simplify the way we handle the state, I think we can get away with one state property and adjust the render method to show the right class